server/offenceCodeDecisions/Placeholder.ts

```diff
@@ -34,31 +34,35 @@ export function getPlaceholderValues(
   answerData?: AnswerData
 ): PlaceholderValues {
   return {
-    prisonerFirstName: convertToTitleCase(prisoner.firstName),
```

Comment:
Moved all of the formatting into the processText function to keep it all together in one place. This method is just to get the data now.